conans/test/utils/tools.py

```diff
@@ -718,6 +719,15 @@ def assert_listed_binary(self, requires, build=False, test=False):
             else:
                 raise AssertionError(f"Cant find {r}-{kind} in {reqs}")
 
+    def assert_in_file_contents(self, filename, contents: List[str], path=None):
+        path = path or self.current_folder
+        file = os.path.join(path, filename)
+        assert os.path.exists(file)
+        with open(file) as f:
+            file_contents = "\n".join(f.readlines())
```

Comment:
It is possible to do ``file_contents = f.read()`` to read the whole file, no need to readlines and join them